install/Install_Controller.php

```diff
@@ -783,10 +783,6 @@ private function action_databasePopulation()
 		}
 		$replaces['{$default_reserved_names}'] = strtr($replaces['{$default_reserved_names}'], array('\\\\n' => '\\n'));
 
-		// If the UTF-8 setting was enabled, add it to the table definitions.
-		if (!empty($databases[$db_type]['utf8_support']))
-			$replaces[') ENGINE=MyISAM;'] = ') ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_general_ci;';
```

Comment:
No, it is not replaced at all.
In the past the queries were written to end in:
```
) ENGINE=MyISAM;
````
but in the new version the queries end in:
```
) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_general_ci
_______________^
```
see the missing semicolon that makes the replace useless?